collections/river-and-lake-ice-extent-s1.yaml

```diff
@@ -0,0 +1,74 @@
+Name: Sentinel 1 River and Lake Ice Extent
```

Comment:
I think the title format for `River and Lake Ice Extent S1+S2`  looks good to avoid confusing with S1

something like: `River and Lake Ice Extent - Sentinel 1`